ChemistryLib/PhreeqcIOData/Output.h

```diff
@@ -24,7 +24,7 @@ class BasicOutputSetups
 public:
     explicit BasicOutputSetups(std::string const& project_file_name,
                                bool const use_high_precision_)
-        : output_file(project_file_name + "phreeqc_.out"),
```

Comment:
And one more time. Strings must not be changed.

Reply:
It was corrected but again the change back was committed to the commit for cpp files of ChemistryLib. I will move the correction to the commit for head files of ChemistryLib.